SilKit/source/services/logging/Logger.cpp

```diff
@@ -80,10 +277,29 @@ class SilKitRemoteSink
 
 Logger::Logger(const std::string& participantName, Config::Logging config)
     : _config{std::move(config)}
-{
+{  
     // NB: do not create the _logger in the initializer list. If participantName is empty,
     //  this will cause a fairly unintuitive exception in spdlog.
-    _logger = spdlog::create<spdlog::sinks::null_sink_st>(participantName);
+    for (auto sink : _config.sinks)
+    {
+        if (sink.type == Config::Sink::Type::Remote)
+        {
+            _loggerRemote = std::make_shared<RemoteLogger>(sink.level, participantName); 
+        }
+        else
+        {
```

Comment:
```suggestion
        {
            // NB: logger gets dropped from registry immediately after creating so that two participant with the same
            // participantName won't lead to a spdlog exception because a logger with this name does already exist.

```

Move the comment up here and ...